io.github.gitahead.Gitahead.json

```diff
@@ -0,0 +1,130 @@
+    "--filesystem=home",
+    /*we use the keyring to store credentials*/
+    "--filesystem=xdg-run/keyring",
+    "--talk-name=org.freedesktop.secrets",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.freedesktop.Flatpak"
+  ],
+  "modules": [
+    "shared-modules/libsecret/libsecret.json",
+    {
+      "name": "libsectre"
+    },
+    "shared-modules/intltool/intltool-0.51.json",
+    {
+      "name": "intltool"
+    },
+    {
+      "name": "git",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm0755 $(which git) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-clang-format) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-cvsserver) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-receive-pack) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-shell) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-upload-archive) ${FLATPAK_DEST}/bin/",
+        "install -Dm0755 $(which git-upload-pack) ${FLATPAK_DEST}/bin/"
+      ]
+    },
+    {
+      "name": "graphviz",
+      "cleanup": [
+        "/share/man",
+        "/share/graphviz/doc"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://gitlab.com/graphviz/graphviz/-/archive/2.47.0/graphviz-2.47.0.tar.gz",
+          "sha256": "4b7f61c7c04f39b67b2c87977633d61d9f72ee6f93ff4376acdce140128431d4"
+        }
+      ]
+    },
+    {
+      "name": "libgnome-keyring",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--enable-vala=no",
+        "--enable-introspection=no"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download-fallback.gnome.org/sources/libgnome-keyring/3.9/libgnome-keyring-3.9.1.tar.xz",
+          "sha256": "8eaca3d4f04ba834acb6c5037b26c3f0837ee4b71ccb380e7810052f7cde381b"
+        },
+        {
+          "type": "file",
+          "path": "gitahead.patch"
+        }
+      ],
+      "cleanup":[
+        "/share/gtk-doc"
+      ]
+    },
+    {
+      "name": "gitahead",
+      "buildsystem": "simple",
+      "cleanup": [
+        "/usr/local/share/",
+        "/usr/local/man",
+        "/usr/local/share/doc",
+        "/usr/local/share/man",
+        "/usr/local/Resources/*.html",
+        "/share/man"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/gitahead/gitahead.git",
+          "commit":"bb747caef098140d9674990b1d79d096d1eac8df"
+        },
+        {
+          "type": "patch",
+          "path": "gitahead.patch"
+        },
+        {
+          "type": "file",
+          "path": "io.github.gitahead.Gitahead.xml"
+        },
+        {
+          "type": "file",
+          "path": "io.github.gitahead.Gitahead.desktop"
+        },
+        {
+          "type": "script",
+          "dest-filename": "run.sh",
+          "commands": [
+            "/app/usr/local/GitAhead"
```

Comment:
Can't you just set the prefix when building instead if doing all that dance with a wrapper script?

Reply:
I tested it originally with the `cmake-ninja` build system and it didn't really work. I'll change it to call the binary directly

Reply:
I tested it with the `cmake-ninja`  build system originally but I couldn't get it to even start configuring to build. I'll change it to call the binary directly.